web/modules/custom/dcamp_sessions/src/Session.php

```diff
@@ -0,0 +1,101 @@
+<?php
+
+namespace Drupal\dcamp_sessions;
+
+use Drupal\Component\Utility\Xss;
+
+/**
+ * Class to interact with session data
+ */
+class Session {
+
+  /**
+   * Positions in sessionsGathered array
+   */
+  const NAME = 2;
+  const DRUPAL_ORG = 3;
+  const PHOTO = 4;
+  const TWITTER = 5;
+  const BIO = 6;
+  const SESSION_TYPE = 7;
+  const SESSION_LEVEL = 8;
+  const LANGUAGE = 9;
+  const TITLE= 10;
+  const DESCRIPTION = 11;
+
+  /**
+   * @var string[]
+   */
+  private $sessionGathered;
+
+  public function __construct( $session_gathered) {
+    $this->sessionGathered = $session_gathered;
+  }
+
+  public function getAuthorsName() {
+    return Xss::filter($this->sessionGathered[$this::NAME]);
+  }
+
+  public function getAuthorsDrupalName() {
+    $user_input = Xss::filter($this->sessionGathered[$this::DRUPAL_ORG]);
+    $url = NULL;
+    if (strpos($user_input, "drupal.org") > 4) {
```

Comment:
I also have this at https://github.com/AsociacionDrupalES/DrupalCampSpain/blob/master/web/modules/custom/dcamp_attendees/src/Entity/IndividualSponsor.php#L120.

Let's join forces and move this utility methods to a separate service.